src/core/or/connection_edge.c

```diff
@@ -597,6 +597,40 @@ connected_cell_format_payload(uint8_t *payload_out,
   return connected_payload_len;
 }
 
+/* This is an onion service client connection: Export the client circuit ID
+ * according to the HAProxy proxy protocol. */
+STATIC void
+export_hs_client_circuit_id(const edge_connection_t *edge_conn,
+                            connection_t *conn)
+{
+  char *buf;
+  const char dst_ipv6[] = "::1";
+  /* See RFC4193 regarding fc00::/7 */
+  const char src_ipv6_prefix[] = "fc00:dead:beef:4dad:";
```

Comment:
Nice.

Reply:
Actually, I'm unsure about this? Is it fine this is a static string or should it be configurable?

Reply:
I would prefer this to be configurable too, but I'm not sure how to do so without adding more torrc options. Maybe an environmental variable?